src/Sarif/CodeGenHints.json

```diff
@@ -412,11 +442,6 @@
       }
     }
   ],
-  "Resources.MessageStrings": [
```

Comment:
>MessageStrings [](start = 13, length = 14)

So we've confirmed that there really never was a use for these "global message strings"? Or rather, I suppose, they were used _only_ in notifications?

Reply:
Oh, I see, this became `Tool.GlobalMessageStrings`.

---
In reply to: [255306444](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255306444) [](ancestors = 255306444)